src/System.Device.Gpio/System/Device/Spi/SpiDevice.cs

```diff
@@ -7,13 +7,23 @@ namespace System.Device.Spi
     /// <summary>
     /// The communications channel to a device on a SPI bus.
     /// </summary>
-    public abstract partial class SpiDevice : IDisposable
+    public abstract class SpiDevice : IDisposable
     {
+        public SpiDevice(SpiConnectionSettings settings, Board board)
```

Comment:
Same here, this is removing the paramterless constructor which we can't take as it is a breaking change.